databases/bookings_db.txt

```diff
@@ -1,16 +1,4 @@
-LennartMantel,How to Train Your James,21:00,
-LennartMantel,Lennart; The Professional,21:00,
-LennartMantel,The Bi-Curious Case of Carl Frohburg,18:00,
-LennartMantel,Lennart; The Professional,21:00,
-LennartMantel,O Sam - Where Art Thou?,19:45,
-LennartMantel,The Bi-Curious Case of Carl Frohburg,18:00,
-LennartMantel,How to Train Your James,21:00,
-l,Lennart; The Professional,13:00,
-l,The Bi-Curious Case of Carl Frohburg,18:00,
-l,Lennart; The Professional,21:00,
-l,How to Train Your James,21:00,
-l,The Bi-Curious Case of Carl Frohburg,11:00,
-SamuelFoley,Screen,21:00,
-SamuelFoley,Screen,11:00,
-SamuelFoley,Screen,16:00,
-SamuelFoley,Screen,13:00,
+l,Screen 7 -  Luke the movie 2,03:00,
+l,Screen 7 -  B Moby Dick,12:00,
+l,Screen 2 -  Literally just lennart,21:00,
```

Comment:
This doesn't seem like the correct format?

Reply:
<img width="375" alt="python_qDAAyYo4Xq" src="https://user-images.githubusercontent.com/37147518/102018819-93786980-3d67-11eb-8ba4-32490a6a466a.png">
Looks fine to me

Reply:
![image](https://user-images.githubusercontent.com/49402982/102018859-d5091480-3d67-11eb-8ac7-de4958ba2d6e.png)

Reply:
my bad wrong db